docs/src/use/configure/rules.md

```diff
@@ -69,76 +73,96 @@ Configuration comments can include descriptions to explain why the comment is ne
  */
 ```
 
-### Using configuration files
+### Using Configuration Files
 
 To configure rules inside of a configuration file, use the `rules` key along with an error level and any options you want to use. For example:
 
-```json
-{
-    "rules": {
-        "eqeqeq": "off",
-        "curly": "error",
-        "quotes": ["error", "double"]
+```js
+export default [
+    {
+        rules: {
+            eqeqeq: "off",
+            "no-unused-vars": "error",
+            "prefer-const": "warn"
```

Comment:
```suggestion
            "prefer-const": ["error", { "ignoreReadBeforeAssign": true }]
```

To have an example with options, since the sentence before this code block mentions options.